pkgs/applications/emulators/rpcs3/default.nix

```diff
@@ -66,8 +65,7 @@ gcc11Stdenv.mkDerivation {
     qtbase qtquickcontrols qtmultimedia openal glew vulkan-headers vulkan-loader libpng ffmpeg
     libevdev zlib libusb1 curl wolfssl python3 pugixml faudio flatbuffers
   ] ++ lib.optional sdl2Support SDL2
-    ++ lib.optional pulseaudioSupport libpulseaudio
-    ++ lib.optional alsaSupport alsa-lib
+    ++ lib.optionals cubebSupport cubeb.passthru.backendLibs
```

Comment:
rpcs3 supports pulseaudio/alsa independently of cubeb (and tries to link with it). Unless the dependency is propagated (and even then I'm a bit iffy 'cause it's less clear), cubeb should be added alongside, not replacing.

Reply:
ALSA and Pulse aren't used independently anymore.
https://github.com/RPCS3/rpcs3/commit/37a722cc1deac48658ced76d251aedcd9a9dac03

Reply:
You're right, it's been a while since I've looked at the code. Am testing this now, will report back.

Reply:
Yep, works fine, lgtm.